babel.config.js

```diff
@@ -304,6 +305,16 @@ module.exports = function (api) {
     ].filter(Boolean),
   };
 
+  if (jestSnapshot) {
```

Comment:
Can we configure Jest to just not load the Babel config for snapshots?

Reply:
https://github.com/jestjs/jest/blob/2c3d2409879952157433de215ae0eee5188a4384/packages/jest-snapshot/src/utils.ts#L363-L368

This doesn't seem to be possible at the moment.

Reply:
To be honest, I even want to open a pr in jest to avoid using `@babel/core`.